src/apps/api/views/submissions.py

```diff
@@ -89,3 +90,17 @@ def upload_submission_scores(request, submission_pk):
         )
 
     return Response()
+
+
+@api_view(('GET',))
+def can_make_submission(request, phase_id):
+    # TODO: Check that user is in competition
```

Comment:
todo to hit, but still TBD on exactly where to do this (re: SAS discussion above)

Reply:
This todo also require implementing `CompetitionParticipant` things I think.

Reply:
Ahh, okay, made #118 to track that